### ISSUE TITLE
how to activate scrolling?

### ISSUE THREAD

**Issue by skeeran** [damiandennis/yii2-scrollingblocks#2]
its working.. but how to activate the infinite scrolling ?

thanks

**Comment by damiandennis** [comment]
currently it requires the items to fill the full page before it will function. I will be changing that in a future release just have not got around to it just yet.

**Comment by skeeran** [comment]
mmm its not working then form me... my page is full but its not load new content when i go to end of the page...

**Comment by damiandennis** [comment]
got an example? should be functional if your items extend beyond the first page.

**Comment by skeeran** [comment]
i am using the example code:

```
ScrollingBlocks::widget([
    'dataProvider' => $dataProvider,
    'itemView' => '_view',
    'wallOptions' => [
      'itemWidth' => 200,
      'offset' => 5,
      'autoResize' => true,
      'flexibleWidth' => '20%', //This can be determined by the responsive widths below.
    ],
    'responsiveWidths' => [
      '(max-width: 520px)'=>'100%',
      '(min-width: 521px) and (max-width: 655px)'=>'50%',
      '(min-width: 656px) and (max-width: 1000px)'=>'33%',
      '(min-width: 1001px) and (max-width: 1199px)'=>'25%',
      '(min-width: 1200px)'=>'20%',
    ]
]); 
```

my view : 

```
<div class="col-sm-3">
    <div style="height: 200px; border: solid 1px #DEDEDE; padding: 10px; margin-bottom: 20px;">
        <?php echo $model->title ?>
    </div>
</div>
```

**Comment by adipaulo** [comment]
Hello,

It's not working for me neither.. ( i'm using the same code as above )

getting this error on inspector 

 Uncaught SyntaxError: Unexpected token ,

on this part

<script type="text/javascript">$(function() {
                yii.scrollingblocks.setup({"id":"w0","input":"#w0 .item","wallOptions":{"itemWidth":200,"offset":5,"autoResize":true,"flexibleWidth":"20%","container":$('#w0')},"size":20,"limit":20,"page":0,"total":2,"pageNumLabel":"page","pageSizeLabel":"per-page","responsiveWidths":[],"callback":,"filterForm":null,"onLoad":});
            });

Any help is appreciated 

Regards,
Adriano

**Comment by adipaulo** [comment]
script type="text/javascript">$(function() {
                yii.scrollingblocks.setup({"id":"w0","input":"#w0 .item","wallOptions":{"itemWidth":200,"offset":5,"autoResize":true,"flexibleWidth":"20%","container":$('#w0')},"size":20,"limit":20,"page":0,"total":2,"pageNumLabel":"page","pageSizeLabel":"per-page","responsiveWidths":[],"callback":,"filterForm":null,"onLoad":});
            });

**Comment by damiandennis** [comment]
fix in 2982606
